apiary_v2.apib

```diff
@@ -1807,12 +2081,143 @@ This endpoint can be used to retrieve a paginated list of the individual jobs as
+                    "artwork_width": 49.29,
+                    "artwork_height": 43.65,
+                    "units": "mm"
+                  }
+                },
+                "previews": [
+                  {
+                    "preview": "https://cdn.unmade.com/E3DC59C3_AB4030F7.png",
+                    "name": "Front"
+                  },
+                  {
+                    "preview": "https://cdn.unmade.com/E3DC59C3_MQOCYBE6.png",
+                    "name": "Left"
+                  },
+                  {
+                    "preview": "https://cdn.unmade.com/E3DC59C3_8NVBFEUD.png",
+                    "name": "Back"
+                  },
+                  {
+                    "preview": "https://cdn.unmade.com/E3DC59C3_NFVDTF54.png",
+                    "name": "Right"
+                  }
+                ],
+                "personalisation": [
+                  {
+                    "name": "number",
+                    "value": "2"
+                  }
+                ],
+                "components": [
+                  {
+                    "name": "Tech White Button",
+                    "display_name": "White",
+                    "type": "tech_button",
+                    "reference": "btn-281"
+                  }
+                ],
+                "changelog": [
+                  {
+                    "field": "ex_factory_date",
+                    "value": "2024-08-05",
+                    "modification": "2024-07-29 13:28:22.519016+00:00",
+                    "user": "vendor@embed.unmade.com"
+                  },
+                  {
+                    "field": "arrive_by_date",
+                    "value": "2024-08-19",
+                    "modification": "2024-07-29 13:28:22.519016+00:00",
+                    "user": "vendor@embed.unmade.com"
+                  }
+                ],
+                "design_element_colours": [
+                  {
+                    "cmyk": [
+                      0.7,
+                      0.7,
+                      0.7,
+                      1.0
+                    ],
+                    "colour_name": "Noir",
+                    "display_colour": "#000000",
+                    "is_spot_colour": false,
+                    "brand_reference": "Noir",
+                    "factory_reference": "blk-13"
+                  },
+                  {
+                    "cmyk": [
+                      1.0,
+                      0.0,
+                      0.9,
+                      0.2
+                    ],
+                    "colour_name": "Lime Green",
+                    "display_colour": "#00cc14",
+                    "is_spot_colour": false,
+                    "brand_reference": "Lime Green",
+                    "factory_reference": "grn-344"
+                  }
+                ]
+              },
+              // ...remaining jobs here
+            ]
+          } 
```

Comment:
each job it's a lot of information .... I wonder if we need to break them into different endpoints (thinking of performance) .... for example the `changelog` could be it's own endpoint .

Also you added `components` `design elements` and roster under `personalisation` 👍🏽  ... not `customised placements` ?

Reply:
This was just taken from the existing endpoint. Nothing is under `personalisation`: I think the objects in this array are just the roster values? I think I'll rename it to roster.

I agree that it feels like a big load of data. Fine to split out changelog, but on balance I think the rest should stay in one response. Or we take out all the "personalisation" related stuff until someone needs it?

Reply:
I've updated this to be more similar to the data in the existing `/manufacturing_files` response which has much less stuff. The job endpoint I was using with design elements etc. in it wasn't a public endpoint

Reply:
We can always add more data if we want to and if it still makes sense from the "payload" side of it. Also not a bad idea of another endpoint to retrieve all the personalisations if they need for their own purposes ... In other words we can keep tweaking them consciously when we see gaps etc